ubuntudesign/maas.io
# Updating copy on get started and updating the MAAS Logo

## Done

- Improved the new MAAS logo
- Adds copy to the get started page

## QA

Pull down the branch and run setup commands, then look at the get started page and check the new logo is loading

## Issue / Card

#35

## Screenshots

_n/a_
